### PR TITLE
[DX-483] G seo technical content

### PR DESCRIPTION
**Pull request by madebyguie**

### CODE REVIEW

File: tyk-docs/content/advanced-configuration/distributed-tracing/newrelic.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 title: "New Relic"
 date: 2022-04-04
 tags: ["distributed tracing","NewRelic"]
-description: "You can create your own tracing implementation using the Trace API. The Trace API is used to send distributed tracing data to New Relic: either in the generic format or the Zipkin data format."
+description: "Create your own tracing implementation with Trace API, used to send distributed tracing data to New Relic, in the generic or Zipkin data format."
```
Comment (letzya) [inline tyk-docs/content/advanced-configuration/distributed-tracing/newrelic.md]: The new text is not clear to me. Can you please check it again

File: tyk-docs/content/advanced-configuration/manage-multiple-environments/with-tyk-on-premises.md [hunk 2]
```diff
@@ -13,9 +13,9 @@ Tyk Gateway has a very powerful functionality that allows you to selectively cho
 
 Imagine the case where you have two sets of APIs, Internal & External.  You want to prevent your Internal APIs from being accessed or visible outside your protected network.  Well, [sharding](/docs/advanced-configuration/manage-multiple-environments/#api-sharding) makes it extremely easy to configure your Tyk Gateways from the Dashboard.
 
-# Instructions
+## Instructions
```
Comment (letzya) [inline tyk-docs/content/advanced-configuration/manage-multiple-environments/with-tyk-on-premises.md:16]: Why is this title in the same level as the one above and not one level down with I.e. ###?

File: tyk-docs/content/release-notes/version-2.4.md [hunk 3]
```diff
@@ -7,19 +7,19 @@ menu:
 weight: 13
 ---
 
-# New in this release:
+## New in this release:
```
Comment (letzya) [inline tyk-docs/content/release-notes/version-2.4.md:10]: Why do we want this change?

**Comment by madebyguie** [comment]
Updated PR fixing conflicts created from latest updates on docs and merge new changes from main.

Mainly duplicated or missing H1 tags (just adapting the header level of some titles), resized large images and fixed some long meta descriptions.

**Review by letzya — CHANGES_REQUESTED** [review]

File: tyk-docs/content/release-notes/version-2.5.md [hunk 4]
```diff
@@ -7,14 +7,14 @@ menu:
 weight: 12
 ---
 
-# <a name="new"></a>New in this Release:
+## <a name="new"></a>New in this Release:
```
Comment (letzya) [inline tyk-docs/content/release-notes/version-2.5.md:10]: Agan, why do we change this?

**Review by letzya — CHANGES_REQUESTED** [review]

File: tyk-docs/content/release-notes/version-3.2.md [hunk 5]
```diff
@@ -43,14 +43,14 @@ In addition, you can now create your own custom permissions using the Additional
 With Go response plugins you are now able to modify and create a full request round trip made through the Tyk Gateway. 
 Find out more about [plugins]({{< ref "plugins" >}}) and how to write [Go response plugins]({{< ref "plugins/supported-languages/golang#using-a-go-response-plugin" >}}).
 
-# Bug fixes and minor changes
+## Bug fixes and minor changes
```
Comment (letzya) [inline tyk-docs/content/release-notes/version-3.2.md:46]: It's on the same level as "what's new" as such We need to change "what's new to ## as well and the other headings after it to ###

File: tyk-docs/content/release-notes/version-3.2.md [hunk 6]
```diff
@@ -43,14 +43,14 @@ In addition, you can now create your own custom permissions using the Additional
...
 
 In addition to the above, version 3.2 includes all the fixes that are part of 3.0.5
 https://github.com/TykTechnologies/tyk/releases/tag/v3.0.5
 
-# Updated Versions
+## Updated Versions
```
Comment (letzya) [inline tyk-docs/content/release-notes/version-3.2.md:51]: same as the last message. it' level #

File: tyk-docs/content/release-notes/version-3.2.md [hunk 7]
```diff
@@ -43,14 +43,14 @@ In addition, you can now create your own custom permissions using the Additional
...
 Tyk Gateway 3.2
 Tyk Dashboard 3.2
 
-# Upgrade process
+## Upgrade process
```
Comment (letzya) [inline tyk-docs/content/release-notes/version-3.2.md:55]: again level 1, why to change it to level 2?

**Review by letzya — CHANGES_REQUESTED** [review]

File: tyk-docs/content/shared/x-tyk-gateway.md [hunk 8]
```diff
@@ -1,6 +1,4 @@
 
-## TYK OAS API Object
```
Comment (letzya) [inline tyk-docs/content/shared/x-tyk-gateway.md:2]: Why do you want to remove this title? 
Also this is an auto-gen doc, we cannot edit it here. If we need to we need ti change the source

File: tyk-docs/static/css/docs.css [hunk 9]
```diff
@@ -2,6 +2,7 @@
   Doc Specific Changes
    ========================================================================== */
 
+.products h1,
```
Comment (letzya) [inline tyk-docs/static/css/docs.css:5]: what does this do?

File: tyk-docs/content/release-notes/version-2.4.md (hunk 3 above)
Comment (letzya) [reply tyk-docs/content/release-notes/version-2.4.md:10]: ok, i understand level 1 looks odd

**Review by letzya — CHANGES_REQUESTED** [review]

File: tyk-docs/static/css/docs.css (hunk 9 above)
Comment (madebyguie) [reply tyk-docs/static/css/docs.css:5]: This is to make sure all the titles in the home page were aligned. 
As it was missing a H1 tag, therefore I've changed what should have been a h1 which is the "Tyk API Management" title. And when it was changed, i've aligned it with the style to the other rules for the h2 tags.

This is an SEO requirement

File: tyk-docs/content/release-notes/version-3.2.md (hunk 7 above)
Comment (madebyguie) [reply tyk-docs/content/release-notes/version-3.2.md:55]: I've updated with the last commit all the titles for this page to be in the levels they should, but leaving only 1 # .

There should be only 1 level 1 in each page for SEO purposes, this is one of most important factors to make clear to google what this page is about.

File: tyk-docs/content/release-notes/version-3.2.md (hunk 6 above)
Comment (madebyguie) [reply tyk-docs/content/release-notes/version-3.2.md:51]: I've updated with the last commit all the titles for this page to be in the levels they should, but leaving only 1 # .

There should be only 1 level 1 in each page for SEO purposes, this is one of most important factors to make clear to google what this page is about.

File: tyk-docs/content/shared/x-tyk-gateway.md (hunk 8 above)
Comment (madebyguie) [reply tyk-docs/content/shared/x-tyk-gateway.md:2]: The title was duplicated, the title for the page where this is used is in x-tyk-oas-doc.md.
This is a shared file within the "content" directory, are you sure is auto-gen doc, if it's within 'content'?

**Review by madebyguie — COMMENTED** [review]
I've updated this line. I think is better written.
The purpose of editing this is to shorten the meta description for SEO purposes

**Review by madebyguie — COMMENTED** [review]
I've updated the titles on this page to make sure that there is no repeated <h1> ( or # title ), as there can be only one, and as there was other h1 on the page I downgraded all the titles accrodingly

File: tyk-docs/content/advanced-configuration/distributed-tracing/newrelic.md (hunk 1 above)
Comment (madebyguie) [reply tyk-docs/content/advanced-configuration/distributed-tracing/newrelic.md]: This has been fixed in the new commit

**Comment by letzya** [comment]
closing till we can work on it. Ticket is updated in the subject

**Pull request closed without merging**